### PR TITLE
Added availability check to use it with iOS7

### PR DESCRIPTION
**Pull request by acecilia**
Fix to use the textField in iOS7 (by downloading and adding the source
code to the project)

### CODE REVIEW

**Comment by RuiAAPeres** [comment]
You can remove the `else {}`.

File: Sources/SkyFloatingLabelTextField.swift [hunk 1]
```diff
@@ -539,7 +539,9 @@ public class SkyFloatingLabelTextField: UITextField {
     
     /// Invoked when the interface builder renders the control
     override public func prepareForInterfaceBuilder() {
-        super.prepareForInterfaceBuilder()
+        if #available(iOS 8.0, *) {
+            super.prepareForInterfaceBuilder()
+        } else {}
```
Comment (gergelyorosz) [inline Sources/SkyFloatingLabelTextField.swift]: +1 on what @RuiAAPeres said. Could you remove this please, and it will be good to go 👍

Comment (acecilia) [inline Sources/SkyFloatingLabelTextField.swift]: Sure! Hell, how I missed that xD

**Comment by gergelyorosz** [comment]
Perfect, thanks for the improvement @acecilia 👍 !

**Comment by intonarumori** [comment]
Can we get some context of why we needed this, can we link an issue?

The preferred way to check API availability is `respondsToSelector` not hardcoding ios versions, is there any reason not to use it here?
https://www.raywenderlich.com/42591/supporting-multiple-ios-versions-and-devices

**Comment by RuiAAPeres** [comment]
@intonarumori that article is from 2013. From [Apple's blog](https://developer.apple.com/swift/blog/?id=29): 

> Availability: Using the latest SDKs ensures you get access to new features and information about platform changes. But sometimes you still need to target an older OS, and Swift makes doing so much easier and safer. The Swift compiler now shows an error when you use an API that is too new for your target OS, and #available blocks can safely wrap lines of code to only run when on the right OS versions.

**Comment by intonarumori** [comment]
@RuiAAPeres Thanks for the link, seems less articulate to me but you are right this is the way forward.

Shouldn't we just use it for the whole method then?
Only having the `super` call in the condition seems weird to me, especially since `super` is required to be called.
Using it on the whole `func` would articulate the intent better imho.

```
@available(iOS 8, *)
override public func prepareForInterfaceBuilder() {
        super.prepareForInterfaceBuilder()
        self.selected = true
        _renderingInInterfaceBuilder = true
        self.updateControl(false)
        self.invalidateIntrinsicContentSize()
    }
```

**Pull request merged**